docs/tutorial/nbwidgets.rst

```diff
@@ -13,31 +13,37 @@ to help you analyze your data. To enable such widgets, start the notebook with::
 Channel slicing
 ---------------
 
-Let's say we want to do some analyses on slices of channel data. It would be nice to just quickly visually select some
-regions using a widget. Let's load the file and run the widget::
+Let's say we want to do some analyses on slices of channel data.
+It would be nice to just quickly visually select some regions using a widget.
+Let's load the file and open the :class:`~lumicks.pylake.nb_widgets.range_selector.SliceRangeSelectorWidget` which we can conveniently open for a slice using the :meth:`~lumicks.pylake.channel.Slice.range_selector()` method::
```

Comment:
This sentence sounds a bit odd. Additionally we don't want to give the impression that the user should construct the widget manually (it's not public through `lk`) Maybe change to
```suggestion
Let's load the file and open an interactive plot using the :meth:`~lumicks.pylake.channel.Slice.range_selector()` method::
```

and then below the codeblock/figure can add
```
This returns a :class:`~lumicks.pylake.nb_widgets.range_selector.SliceRangeSelectorWidget` ....
```

Reply:
Yeah, that flows better.